infrastructure/main.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = "<= 1.0.0"
+  required_version = "~= 1.0"
```

Comment:
Better update it like `>= 1.0.0`. That newer versions can be supported.

I tested with `1.1.8` and all is working.